code/game/objects/items/weapons/chaplain_items.dm

```diff
@@ -0,0 +1,110 @@
+/obj/item/weapon/nullrod
+	name = "null rod"
+	desc = "A rod of pure obsidian, its very presence disrupts and dampens the powers of paranormal phenomenae."
+	icon_state = "nullrod"
+	item_state = "nullrod"
+	slot_flags = SLOT_BELT
+	force = 15
+	throw_speed = 1
+	throw_range = 4
+	throwforce = 10
+	w_class = ITEMSIZE_SMALL
+	drop_sound = 'sound/items/drop/sword.ogg'
+	pickup_sound = 'sound/items/pickup/sword.ogg'
+	var/cooldown = 0 // floor tap cooldown
+
+	suicide_act(mob/user)
+		var/datum/gender/T = gender_datums[user.get_visible_gender()]
+		to_chat(viewers(user),"<span class='danger'>[user] is impaling [T.himself] with the [src.name]! It looks like [T.he] [T.is] trying to commit suicide.</span>")
+		return (BRUTELOSS|FIRELOSS)
+
+/obj/item/weapon/nullrod/staff
+	name = "null staff"
+	desc = "A staff of pure obsidian, its very presence disrupts and dampens the powers of paranormal phenomenae."
+	icon_state = "nullstaff"
+	item_state = "nullstaff"
+	slot_flags = SLOT_BELT | SLOT_BACK
+	w_class = ITEMSIZE_LARGE
+
+/obj/item/weapon/nullrod/orb
+	name = "null sphere"
+	desc = "An orb of pure obsidian, its very presence disrupts and dampens the powers of paranormal phenomenae."
+	icon_state = "nullorb"
+	item_state = "nullorb"
+
+/obj/item/weapon/nullrod/athame
+	name = "null athame"
+	desc = "An athame of pure obsidian, its very presence disrupts and dampens the powers of paranormal phenomenae."
+	icon_state = "nullathame"
+	item_state = "nullathame"
+
+/obj/item/weapon/nullrod/verb/change(mob/user)
+	set name = "Reconfigure Null Item"
+	set category = "Object"
+	set src in usr
+
+	var/list/options = list()
```

Comment:
You can initialize the list instead of creating an empty one and then adding all of the indices like so:
```
var/list/options = list(
    "Null rod" = /obj/item/weapon/nullrod,
    "Null staff" = /obj/item/weapon/nullrod/staff,
    "Null sphere" = /obj/item/weapon/nullrod/orb,
    "Null athame" = /obj/item/weapon/nullrod/athame
)
```
I don't believe the internal lists are necessary here, as each of these only contains a single path.